docs/user/exercises/modeling.rst

```diff
@@ -1,2 +1,193 @@
 Modeling exercise
-=================
\ No newline at end of file
+=================
+.. contents:: Content of this document
+    :local:
+    :depth: 2
+
+
+Overview
+--------
+
+Conducting a Modeling exercise consists of 3 steps:
+
+1. **Instructor prepares exercise:** Creates and configures the modeling exercise in Artemis.
+2. **Student solves exercise:** Student works on the exercise and submits the solution
+3. **Instructor assesses submissions:** Reviews the submitted exercises and creates results for the students.
+
+Setup
+--------
+
+The following sections describe the supported features and the process of creating a new modeling exercise.
+
+- Open |course-management|
+- Navigate into **Exercises** of your preferred course
+
+    .. figure:: modeling/course-management-course-dashboard.png
+              :align: center
+
+- Click on **Create new modeling exercise**
+
+    .. figure:: modeling/create-new-modeling-exercise.png
+              :align: center
+
+Following screenshot illustrates the first section of the form. It consists of:
+
+- *Title*: Title of an exercise
+- *Categories*: Category of an exercise
+- *Difficulty*: Difficulty of an exercise. (No level, Easy, Medium or Hard)
+- *Mode*: Solving mode of an exercise. *This cannot be changed afterwards* (Individual or Team)
+- *Release Date*: Date after which students can access the exercise
+- *Due Date*: Date till when students can access the exercise
+- *Assessment Due Date*: Date till when instructors can assess the exercise submitted by students
+- *Inclusion in course score calculation*: Option that determines whether or not to include exercise in course score calculation
+- *Points*: Total point of an exercise
+- *Bonus Points*: Bonus points of an exercise
+- *Diagram Type*: Type of diagram that is used throughout an exercise
+
+    .. figure:: modeling/create-modeling-exercise-form-1.png
+              :align: center
+
+.. note::
+   Fields marked with red are mandatory to be filled.
+
+.. note::
+   - Field **Diagram Type** determines the components that students/instructors can use while working on the exercise.
+   - For example: If the instructor selects Class diagram as its Diagram Type, users (instructors/students) will now only be able to use components of class diagram throughout the exercise.
+
+    .. figure:: modeling/class-diagram-diagram-type.png
+              :align: center
+
+Following screenshot illustrates the second section of the form. It consists of:
+
+- *Enable automatic assessment suggestions*: Option that determines whether or not to enable assessment suggestions automatically
+- *Problem Statement*: Field to state problem statement of an exercise
+- *Assessment Instructions*: Instructions for instructor while assessing the submission
```

Comment:
Fine for now, should be changed to a link to the dedicated assessment documentation page once it exists.